app/config/FrontendAppConfig.scala

```diff
@@ -106,8 +106,7 @@ class FrontendAppConfig @Inject() (val configuration: Configuration,
   lazy val deploymentNotification : Boolean =
     configuration.get[Boolean]("microservice.services.features.deployment.notification.enabled")
 
-  lazy val declarationEmailEnabled: Boolean =
-    configuration.get[Boolean]("microservice.services.features.declaration.email.enabled")
+  lazy val declarationEmailEnabled: Boolean = true
```

Comment:
Is this perhaps a local change that shouldn't be committed?

Reply:
yup I didnt meant to include this, was trying to get the option part of the address.